synthesizer/process/src/stack/finalize_types/mod.rs

```diff
@@ -157,6 +158,17 @@ impl<N: Network> FinalizeTypes<N> {
                         None => bail!("'{identifier}' does not exist in struct '{struct_name}'"),
                     }
                 }
+                // Access the member on the path to output the register type.
+                (FinalizeType::Plaintext(PlaintextType::ExternalStruct(locator)), Access::Member(identifier)) => {
+                    // Retrieve the member type from the struct and check that it exists.
+                    let external_stack = stack.get_external_stack(locator.program_id())?;
```

Comment:
Note. Suppose that first access returns a struct and then a second access goes to an external struct in that one. e.g `A -> B -> C`. It must be the case that program `A` also imports program `C`. This is fine, just something to keep in mind on the Leo side.